pkg/newreleases/newreleases.go

```diff
@@ -152,7 +152,7 @@ func ExclusionSliceFromCfg(exclusions []config.NewReleasesExclusion) []newreleas
 func ProjectToCfg(project newreleases.Project) config.NewReleasesProject {
 	return config.NewReleasesProject{
 		Name:               project.Name,
-		EmailNotification:  string(project.EmailNotification),
+		EmailNotification:  emailNotificationToString(project.EmailNotification),
```

Comment:
Suggest creating a new type in pkg/config, and take inspiration from `pkg/config/loglevel.go` on parsing.

Then move the `emailNotificationFromString` function into there as `ParseEmailNotification`

And move `emailNotificationToString` to be a `ReleaseEmailNotification.String()` method on the type.